contracts/clients/test/index.test.ts

```diff
@@ -8,7 +8,7 @@ import { initBlsWalletSigner, Bundle, Operation } from "../src/signer";
 
 import Range from "./helpers/Range";
 
-const domain = arrayify(keccak256("0xfeedbee5"));
+const domain = arrayify("0x425bd46b7016e0395c00f2e05fd74f938023d31f355d5a62fb9c63756c6a5d87");
```

Comment:
This and the value above should be pulled directly from the  `VerificationGateway` (VG) linked to a wallet rather than hardcoded. It is possible someone may:
1. Deploy a VG to a different address if they use a different deployer which would cause this hash to change.
2. Upgrade the VG a wallet uses to a newer/different version, which could completely change the domain for signing. This is a risk currently with how we have the client code.

This requires us to change points consuming `domain` or `defaultDomain` to pull an async value from the wallet which is a larger change set but I think ultimately beneficial.